var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -65,8 +65,6 @@ class Vtk(CMakePackage):
     with when("+python"):
         # Depend on any Python, add bounds below.
         extends("python@2.7:", type=("build", "run"))
```

Comment:
This could probably just be `extends("python")` now correct?

Reply:
Technically, it supports all versions of Python that Spack supports, so we could just get rid of all version specifications. There are thousands of packages where this is true, so I haven't bothered to update all of them.